pkgs/tools/filesystems/sasquatch/default.nix

```diff
@@ -45,7 +45,10 @@ stdenv.mkDerivation rec {
   postPatch = ''
     # Drop blanket -Werror to avoid build failure on fresh toolchains
     # like gcc-11.
-    substituteInPlace squashfs-tools/Makefile --replace ' -Werror' ' '
+    # Add -fcommon as a woarkaround for compiler that default to -fno-common
+    # like upstream gcc-10 or clang-11. Will be fixed upstream at:
+    #   https://github.com/devttys0/sasquatch/pull/44
+    substituteInPlace squashfs-tools/Makefile --replace ' -Werror' ' -fcommon'
```

Comment:
Can we fetch the patch?

Reply:
Unfortunately not directly. `nixpkgs`'s `sasquatch` if a fork by @cole-h with a very invasive https://github.com/devttys0/sasquatch/pull/40 applied (changes every hunk offset).

The repository format is already storing a patch (and not just the source) which makes any (even trivial) changes to it effectively non-composable.